### ISSUE TITLE
Tap console logging multiple times

### ISSUE THREAD

**Issue by vbudhram** [mozilla/fxa-auth-server#1353]
Super low priority, but noticed when running tests locally, multiple of the same lines are printed out. 

<img width="1035" alt="screen shot 2016-07-19 at 3 55 21 pm" src="https://cloud.githubusercontent.com/assets/1295288/16964448/75884a4a-4dc9-11e6-9280-56c64540ea0e.png">

I believe this was introduced with the new version of tap. It does not happen in the fxa-auth-mysql-db which also uses the new version of tap.

**Comment by vladikoff** [comment]
> It does not happen in the fxa-auth-mysql-db which also uses the new version of tap.

Yeah that's so weird, I guess some other scripts that run before it or the `glob` matcher

**Comment by vladikoff** [comment]
Blocked on new tap release..
